src/ucan.ts

```diff
@@ -49,7 +25,7 @@ export const did = async (): Promise<string> => {
  * `rsc`, Resource, the involved resource.
  *
  */
-export const ucan = async ({
+export const compose = async ({
```

Comment:
`compose` is a confusing name here IMO. It's more of a constructor than a composition of multiple UCANs, no?

Reply:
Perhaps: `init`, `build`, or make it a class and call `new`?

Reply:
Yeah, couldn't figure out the perfect name.
Which one would you pick?
- [ ] build
- [ ] create
- [ ] construct
- [ ] make

Reply:
Renamed it to `build` 👍